### ISSUE TITLE
Chemisty suggestion: Kobold fluid replacer device

### ISSUE THREAD

**Issue by Slayerhero90** [naelstrof/KoboldKare#143]
This would be a device (or pair/trio of devices) built in the late Star progression which would allow a kobold strapped to the device to change the fluid produced in their tits, balls, and/or hearts using a very large quantity of a single fluid in a vat attached to the machine. You might also devise and require some fuel or catalyzing agent. Use cases might include:
- Sterilizing scenery kobolds with water so that you don't have to deal with their eggnancies
- Producing a non-farmed supply of fruit juice
- Producing a non-cooked supply of an advanced food such as pie
- Jizzing pure potassium into the river to spite god

It would be very powerful, of course, hence why I think it should be late-Stars or post-Stars content.

**Comment by naelstrof** [comment]
Yeah that sounds really fun, I still have a few things in the current pipeline of stuff that needs to be ironed out-- but after that's done it'd be fun to figure a way to make this rare and engaging.

**Comment by GreenboxGames** [comment]
A sterilizing feature would probably also make the kobalds work as cum containers as they make pretty decent scaleable containers. Perhaps also digestion rejection genes for certain things as well? They can hold a lot, not sure how pumped they can get quite yet as I lose interest around 500-600 units. But this sounds like turning them into some of the tools I once thought would be machines or larger containers, and reagent factories, very similar to how scientists genetically alter bacteria to poop out drugs and such. I really like the sound of this kinda thing. Can't wait to see late-stage genetic mechanics in vanilla play and mods!
